modules/postman-templating/src/template-client.ts

```diff
@@ -23,10 +23,27 @@ export class TemplateClient {
     this.lineBreak = lineBreak || '<br />'
 
     if (allowedImageSources) {
+      this.validateImageSources(allowedImageSources)
       this.xssOptions = filterImageSources(this.xssOptions, allowedImageSources)
     }
   }
 
+  /**
+   * Validate image sources
+   * @param sources
+   */
+  validateImageSources(sources: Array<string>): void {
```

Comment:
I only did non-empty strings checks for the list of sources passed in. Not sure if that's sufficient or the validation should be more thorough (check valid hostname string, etc). 

If only non-empty string check is required, alternatively, the type of `allowedImageSources` can be further restricted to something like `allowedImageSource?: [string, ...string[]]` to use compile time checks. Let me know which is preferred!

Reply:
didnt you check for valid hostname in safeAttrValue